publications/abstracts/reservoir-computing/introduction.tex

```diff
@@ -0,0 +1,28 @@
+\section{Introduction}
+
+Load following has given natural gas an economic edge over nuclear power
+because unlike nuclear,
+natural gas plants can follow grid demand and even shut off when renewable
+penetration makes the
+price of electricity go negative \cite{keppler_carbon_2011}. Advanced reactor
+designs, like Molten Salt
+Reactors (MSR), promise strong load following capabilities due to harder
```

Comment:
use the acros package rather than hard coding the acronym expansion. In this case, \glspl{MSR}